pyannote/audio/tasks/segmentation/multilabel.py

```diff
@@ -251,10 +254,25 @@ def validation_step(self, batch, batch_idx: int):
 
         # mask (frame, class) index for which label is missing
         mask: torch.Tensor = y_true != -1
-        y_pred = y_pred[mask]
-        y_true = y_true[mask]
+        y_pred = y_pred[mask].reshape(shape)
```

Comment:
This will break as soon as `mask` contains at least one `-1` because the overall size of `y_pred` will then be smaller.